### PR TITLE
[TN-9] Listings page (tabs, table and pagination)

### PR DESCRIPTION
**Pull request by RyanFriedman36**
* **Please check if the PR fulfills these requirements**
- [x] The commit message follows our guidelines. For the guidelines check the README.
- [ ] Tests for the changes have been added (for bug fixes / features). As we discussed this is optional but encouraged.

* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)
   - The listings page now was a component with two tabs. one for rentals and one for for sale. In side each tab we have a table filled with dummy data illustrating our layout for the table. There is pagination at the bottom.

* **What is the current behavior?** (You can also link to an open issue here)
   - The listings page just has a phrase noting as such

* **What is the new behavior (if this is a feature change)?**
    - We can view tables of data, switch between the two tabs, and change how many rows are shown in each page (and switch pages)

* **Does this PR introduce a breaking change?** (What changes might users need to make in their application due to this PR?)
  - No

* **Other information**:

### CODE REVIEW

File: src/components/Listings/ListingsTabs/ForSaleListings.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
+import React, { Component } from 'react';
+import DataTable from "../ListingsTable/ListingsTable";
+// import Styles from "../ListingsTable/ListingsTableStyles";
+
+class ForSaleListings extends Component {
+
+  render(){
+    return (
+      <div><DataTable listingType="forSale"/></div>
```
Comment (RyanFriedman36) [inline src/components/Listings/ListingsTabs/ForSaleListings.js:9]: Just FYI: listingType is to let us know which API endpoints to use

File: src/components/NavigationBar/NavigationBarStyles.js [hunk 2]
```diff
@@ -18,4 +18,4 @@ const Styles = styled.div`
   }
 `;
 
-export default Styles;
```
Comment (RyanFriedman36) [inline src/components/NavigationBar/NavigationBarStyles.js:21]: I used CTRL+X to cut this line and paste it somewhere else; there is no change here.

File: src/components/Listings/ListingsTabs/ListingsTabs.js [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+              </Row>
+            </TabPane>
+          </TabContent>
+        </div>
+      );
+    }
+  }
+
+
+  export function ListingsTabs(){
```
Comment (csjoblinksreddit) [inline src/components/Listings/ListingsTabs/ListingsTabs.js:61]: Export is done twice for this function. Once here

File: src/components/Listings/ListingsTabs/ListingsTabs.js [hunk 4]
```diff
@@ -0,0 +1,76 @@
...
+            <Form className="form">
+            <Col>
+              <Tabs />
+            </Col>
+          </Form>
+    </Container>
+    );
+}
+
+export default ListingsTabs;
```
Comment (csjoblinksreddit) [inline src/components/Listings/ListingsTabs/ListingsTabs.js:76]: And again here.

File: src/components/Listings/ListingsTable/PaginationStyle.js [hunk 5]
```diff
@@ -0,0 +1,11 @@
+import { makeStyles } from '@material-ui/core/styles'
```
Comment (csjoblinksreddit) [inline src/components/Listings/ListingsTable/PaginationStyle.js:1]: Lets discuss how to transform this into a .css file instead

File: src/components/Listings/ListingsTable/ListingsTableStyles.js [hunk 6]
```diff
@@ -0,0 +1,15 @@
+import { makeStyles } from '@material-ui/core/styles'
```
Comment (csjoblinksreddit) [inline src/components/Listings/ListingsTable/ListingsTableStyles.js:1]: Lets discuss how to transform this into a .css file instead

File: src/components/Listings/ListingsTable/ListingsTable.js [hunk 7]
```diff
@@ -0,0 +1,169 @@
+import TableHead from '@material-ui/core/TableHead';
```
Comment (csjoblinksreddit) [inline src/components/Listings/ListingsTable/ListingsTable.js:1]: This is a very big file to digest. Lets talk about how we can split this into smaller more digestible files.

**Comment by csjoblinksreddit** [comment]
[Heres](https://imgur.com/a/aW6zpJf) what I got and then the changes I request are 

1. split up listings table.js
2. create a new directory named listings table pagination which includes stylesheet and pagination function from table.js
3. rename tables.js function from datatable to somthing else
4. Listing tabs might be too abstracted. Put the ListingTabs function into the listings.js
5. Put the forsale and rental listing inside of a listing type directory. Can we rename the forsale and rental listings to either forsale and for rent of salelistings and rentalistings
6. Rename tabs class component to ListingTabs and import it inside of listings.js

**Comment by csjoblinksreddit** [comment]
Great job Ryan this looks great! One last thing I would ask is if you could change the styling sheets to .css files instead of .js files. That way we have a uniform way of styling our application. [Here](https://imgur.com/a/u6LGq9p) is an example of how to make that work.

File: frontend/src/components/Listings/ListingsTabs/ListingsType/ForSaleListings.js [hunk 8]
```diff
@@ -0,0 +1,13 @@
+import React, { Component } from 'react';
+import DataTable from "../../ListingsTable/ListingsTable/ListingsTable";
+
+class ForSaleListings extends Component {
```
Comment (csjoblinksreddit) [inline frontend/src/components/Listings/ListingsTabs/ListingsType/ForSaleListings.js:4]: Does this have to be a class component or can it be a functional?

File: frontend/src/components/Listings/ListingsTabs/ListingsType/ForRentListings.js [hunk 9]
```diff
@@ -0,0 +1,13 @@
+import React, { Component } from 'react';
+import DataTable from "../../ListingsTable/ListingsTable/ListingsTable";
+
+class ForRentListings extends Component {
```
Comment (csjoblinksreddit) [inline frontend/src/components/Listings/ListingsTabs/ListingsType/ForRentListings.js:4]: Class or functional component?

File: frontend/src/components/Listings/ListingsTabs/ListingsType/ForSaleListings.js (hunk 8 above)
Comment (RyanFriedman36) [reply frontend/src/components/Listings/ListingsTabs/ListingsType/ForSaleListings.js]: Theres no state they should be functional, just updated

**Pull request merged**